docs/self-managed/platform-deployment/docker.md

```diff
@@ -35,6 +35,16 @@ This will give you a single broker node with the following ports exposed:
 - `26501`: Command API (internal, gateway-to-broker)
 - `26502`: Internal API (internal, broker-to-broker)
 
+### Multi-platform support
```

Comment:
Do the H2 and H3 headers in this doc look correct to you?

Reply:
not sure I follow 😅

H3 like this look fine to me:
![image](https://user-images.githubusercontent.com/209518/214607558-04f8602c-668f-419f-8cf2-4ba050f81792.png)
in contrast to that H2 is also bigger than H3 and smaller than the page name being h1.
![image](https://user-images.githubusercontent.com/209518/214607818-5f7dc1b0-6819-43fb-8a21-d4577c9d40c0.png)

Reply:
Just making sure everything is outlined and nested correctly 👍